subgraphs/uniswap-forks/protocols/sushiswap/src/common/handlers/handleRewardV2.ts

```diff
@@ -0,0 +1,91 @@
+import { BigDecimal, BigInt, ethereum, log } from "@graphprotocol/graph-ts";
+import { NetworkConfigs } from "../../../../../configurations/configure";
+import { MasterChefV2Sushiswap } from "../../../../../generated/MasterChefV2/MasterChefV2Sushiswap";
+import { LiquidityPool, _HelperStore } from "../../../../../generated/schema";
+import { BIGINT_FIVE, BIGINT_ZERO, INT_ZERO, UsageType, ZERO_ADDRESS } from "../../../../../src/common/constants";
+import { getOrCreateToken } from "../../../../../src/common/getters";
+import { findNativeTokenPerToken, updateNativeTokenPriceInUSD } from "../../../../../src/price/price";
+import { getRewardsPerDay } from "../../../../../src/common/rewards";
+
+export function handleRewardV2(event: ethereum.Event, pid: BigInt, amount: BigInt, usageType: string): void {
+  let masterChefPool = _HelperStore.load(pid.toString());
+  let poolContract = MasterChefV2Sushiswap.bind(event.address);
+
+  // Create entity to track masterchef pool mappings
+  if (!masterChefPool) {
+    masterChefPool = new _HelperStore(pid.toString());
+    let getlpAddress = poolContract.try_lpToken(pid);
+    let lpTokenAddress = ZERO_ADDRESS;
+    if (!getlpAddress.reverted) {
+      lpTokenAddress = getlpAddress.value.toHexString();
+    }
+    masterChefPool.valueString = lpTokenAddress;
+    masterChefPool.valueBigInt = BIGINT_ZERO;
+    masterChefPool.save();
+  }
+
+  // Return if pool does not exist - Banana tokens?
```

Comment:
Banana tokens?

Reply:
There is a masterchef staking pool for Banana tokens. It does not exist as a liquidity pool, because it is not an LP token. I was not sure what to do about it since the staking pools in the masterchef are usually associate with a liquidity pools, because you are staking the LP token.